### PR TITLE
feat: initial state for discounts modal

### PR DESCRIPTION
**Pull request by JFrankfurt**
**What changed? Why?**
Added this discounts modal
image

**Notes to reviewers**
future work (documented in asana):

- [ ] add tooltips once we have copy
- [ ] add a state for indicating an `active` discount

**How has it been tested?**
manually

### CODE REVIEW

File: apps/bridge/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "@bugsnag/plugin-react": "^7.19.0",
     "@eth-optimism/contracts-ts": "^0.16.2",
     "@eth-optimism/indexer-api": "^0.0.4",
-    "@headlessui/react": "latest",
+    "@headlessui/react": "^1.7.19",
```
Comment (JFrankfurt) [inline apps/bridge/package.json:16]: `latest` is now a major version up (2.x.x) and doesn't have the same api surface that `bridge` is expecting.

File: apps/web/src/components/Layout/Nav/DesktopNav.tsx [hunk 2]
```diff
@@ -124,9 +124,10 @@ function IconLink({
 
 function DesktopNav({ color }: DesktopNavProps) {
   return (
-    <div className="hidden h-full w-fit flex-grow flex-row items-center items-center justify-between lg:flex">
```
Comment (JFrankfurt) [inline apps/web/src/components/Layout/Nav/DesktopNav.tsx:127]: removing a duplicate classname

File: apps/web/tailwind.config.js [hunk 3]
```diff
@@ -32,17 +32,18 @@ module.exports = {
       ocsblue: 'rgba(0, 82, 255, 1)',
       ocspink: '#FF8DCF',
       ocsyellow: '#FEE002',
+      illoblack: '#0A0B0D',
```
Comment (JFrankfurt) [inline apps/web/tailwind.config.js:35]: from ocs

File: apps/web/pages/usernames.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+          content="Base is a secure, low-cost, builder-friendly Ethereum L2 built to bring the next billion users onchain."
+          name="description"
+        />
+      </Head>
+      <main className="flex w-full flex-col items-center bg-white">
+        <input type="text" placeholder='search for a username'/>
+        <button type='button' onClick={selectName}>option: name1</button>
+
+        {progress === ClaimProgression.CLAIM && <p>
+          unlock your username for free! <button type='button' className="underline" onClick={toggleModal}>learn more</button>
```
Comment (JFrankfurt) [inline apps/web/pages/usernames.tsx:34]: This is placeholder content that will be addressed when we build a real search UI.

**Review by brendan-defi — APPROVED** [review]
lgtm

**Comment by 0xchiaroscuro** [comment]
Screenshot 2024-06-13 at 2 10 52 PM

**Comment by 0xchiaroscuro** [comment]
TOOLTIP

**Comment by cb-heimdall** [comment]
<h1>
🟡
Heimdall Review Status
</h1>
<table>
<tr>
<th>Item</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Minimum Required Reviews</td>
<td>
🟡
0/1
</td>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculations</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
<a href="http://go/heimdall-features">From <code>.codeflow.yml</code></a>
</td>
<td>1</td>
</tr>
<tr>
<td>
<a href="http://go/heimdall-additional-review-requirements">Additional review requirements</a>
<details>
<summary>Show calculations</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
<a href="http://go/heimdall-codeowners">From <code>CODEOWNERS</code></a>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>



<hr>
<strong>
<a href="https://docs.cbhq.net/infra/heimdall">📝 Heimdall docs</a>
·
<a href="https://heimdall-public.cbhq.net/v1/code_change?repository=base-org/web&ref=a7a1231829e1960ad69fe6a32bc8a036dc39a464">🕵️ Advanced details</a>
·
<a href="https://heimdall-public.cbhq.net/v1/pull_request/stuck?repository=base-org/web&pr_number=559">♻️ Refresh Heimdall</a>
·
<a href="https://coinbase.slack.com/archives/ask-heimdall">#️⃣ #ask-heimdall</a>
</strong>

**Pull request merged**